examples/timeouts/test.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+const test = require('ava');
+
+const {delay} = require('.');
```

Comment:
What's the Node.js version used in Stackblitz? With 16 we can use `timers/promises`: https://nodejs.org/docs/latest-v16.x/api/timers.html#timers_timers_promises_api

Reply:
We're currently on `14.x` unfortunately.